crates/wdl-analysis/src/handlers/goto_definition.rs

```diff
@@ -445,20 +468,46 @@ fn resolve_access_expression(
     // `SyntaxKind::AccessExprNode` in the `resolve_by_context` before
     // calling this function.
     let access_expr = wdl_ast::v1::AccessExpr::cast(parent_node.clone()).unwrap();
-    let (target_expr, member_ident) = access_expr.operands();
+    let (target_expr, variant_ident) = access_expr.operands();
```

Comment:
Nit: perhaps call the latter `access_ident` as it's not necessarily a "variant" either (nor "member").